src/index.html

```diff
@@ -1,15 +1,153 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="field">
+          <label class="label" for="surname">Surname:</label>
+          <input
+           type="text"
+           id="surname"
+           name="Surname"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="name">Name:</label>
+          <input type="text"
+           id="name"
+           name="name"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="old">How old are You?</label>
+          <input type="number"
+           id="old"
+           name="old"
+           value="12"
+           min="6"
+           max="100"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="age">Full date of birth:</label>
+          <input type="date"
+           id="age"
+           name="age"
+          >
+        </div>
+        <div class="field">
+          <label class="label" for="box">I accept the term of the agreement</label>
+          <input type="checkbox"
+           id="box"
+           name="box"
+          >
```

Comment:
1 attribute per line, fix everywhere

```suggestion
          <input
            type="checkbox"
            id="box"
            name="box"
          >
```